### ISSUE TITLE
Exception styling

### ISSUE THREAD

**Issue by SeymurMammadrza** [ColdTea-Projects/verborum_ms#5]
https://github.com/ColdTea-Projects/verborum_ms/blob/024d292409436d6a74210402a0efcd73d1222f33/ms_dictionary/src/main/java/de/coldtea/verborum/msdictionary/common/exception/InvalidUUIDException.java#L3

Serial kullan instance için kanka. 

public class MappingException extends RuntimeException {

    @Serial
    private static final Long serialVersionUID = 13242162415L;

    public MappingException(String message) {
        super(message);
    }
}

Burdaki gibi. Ve exceptionları spesifik yapmana gerek yok , IllegalArgumentException filana extend etmeye yani. Direk RuntimeException-a extend et

**Comment by ColdTea-Projects** [comment]
ikinci dedigine katilmiyorum acikcasi belki bu ornekte cok bariz ama genel olarak iliskili tiplerden extend etmek kod anlasilabilirligi acisindan onemli, illegal argument exception in kendi klasina girersen onun da runtime exceptiondan farki olmadigini gorebilirsin, en onemli degisiklik isim, ki kategorize edilebilsin kodu okuyan tarafindan.
serial i neden kullaniyoruz?(bunu cidden bilmedigimden soruyorum), gene IllegalArgumentException'in icinde de kullanilmis, tam sebebini biliyo musun?

**Comment by SeymurMammadrza** [comment]
Daha spesifik yapmak istiyorsan tabi senin bileceğin iş. RuntimeException parent classi diye söyledim. Gerçi orda direk yap demişim, my bad.
Telegrama yazdığım (üşendim chatgptye yazdırdım) şeyi buraya da atıyorum kayıt olsun diye.

Explanation:

serialVersionUID Field:

serialVersionUID is a special field in Java that provides a version identifier for the Serializable class. It is used during the deserialization process to verify that the sender and receiver of a serialized object have loaded classes for that object that are compatible with respect to serialization.
If the sender and receiver have different versions of the class (different serialVersionUID), it may lead to an InvalidClassException during deserialization.
@Serial Annotation:

The @Serial annotation is used in the code to indicate that the serialVersionUID field is managed explicitly. This is not required, but it helps suppress compiler warnings related to the serialization process.
By explicitly providing a serialVersionUID and using @Serial, the developer takes control over the versioning of the class during serialization. It can be useful in situations where you want to manage compatibility across different versions of your application.
In summary, the @Serial annotation in this context is used to indicate that the serialVersionUID field in the CardNotFoundException class is explicitly managed by the developer. This is a practice to take control over the serialization versioning process and avoid potential compatibility issues when the class is serialized and deserialized in different environments or versions of the application.

**Comment by SeymurMammadrza** [comment]
Biz genelde pek uğraşmıyoruz burda, Runtime yapıp geçiyoruz. Unibankda senin yaptığın gibi yapıyorduk
